apps/prairielearn/src/sprocs/check_assessment_access.sql

```diff
@@ -133,7 +188,8 @@ BEGIN
 
         next_active_time = format_date_full_compact(next_active_start_date, display_timezone);
     END IF;
-
+    END IF;
```

Comment:
If you're adding an additional `IF` around existing code, you should indent it to make it easier to read. Unfortunately, Prettier can't yet automatically format queries, so manual formatting is required.

Reply:
@vikramrkini as far as I can tell, this is still unaddressed - do you plan to adjust this?